packages/edit-post/src/components/sidebar/post-trash/index.js

```diff
@@ -1,15 +1,15 @@
 /**
  * WordPress dependencies
  */
-import { PanelRow } from '@wordpress/components';
 import { PostTrash as PostTrashLink, PostTrashCheck } from '@wordpress/editor';
+import { FlexItem } from '@wordpress/components';
 
 export default function PostTrash() {
 	return (
 		<PostTrashCheck>
-			<PanelRow>
+			<FlexItem isBlock>
```

Comment:
It feels like these "FlexItem" elements should move into the `PostStatus` component instead of keeping them within the reusable `PostTrash` and `PostSwitchToDraftButton` components?

Reply:
These are not really reusable, I think — it already had a "PanelRow". Moving the flex item to PostStatus means lifting up the logic for display or not displaying each of these, otherwise you end up with an empty flex element that still takes space

Reply:
Ok 👍